### ISSUE TITLE
History datepicker enables/disables constantly

### ISSUE THREAD

**Issue by Fusseldieb** [home-assistant/home-assistant-polymer#2451]
On HA 0.85.0, when I'm acessing the History panel, the datepicker begins to enable/disable itself every 2-3 seconds and don't stop. I've included just a minimal amount of items and it still flicker/blinks around - it never stops.

Selecting the datepicker is almost impossible, because as soon as it's selected, it disables/enables itself and closes the dropdown again.

I'm observing this issue in Chrome on Windows+Android and Microsoft Edge on Windows.

The Chrome Console reports no error...

![bughist](https://user-images.githubusercontent.com/4715129/51057347-81667d00-15cc-11e9-8e52-5cebcd03ec46.gif)
_(GIF of the issue)_

**Comment by balloob** [comment]
It's disabled when it is loading data. Unclear to me why it would load data all the time, maybe when any state changes?

**Comment by Fusseldieb** [comment]
> maybe when any state changes?

Those entities in the GIF above don't update their values with much frequency, so I doubt that it's this. They update their state every 1 second, but their value stay the same for entire minutes sometimes. 

The disabling of that datepicker should be disabled, so that it doesn't interfere someone trying to choose something.

**Comment by Fusseldieb** [comment]
Bump. @balloob I think I figured out why it does this above. It tries to load the same state over and over again, but why exactly I don't know.
![image](https://user-images.githubusercontent.com/4715129/51292430-d271d900-19f1-11e9-9c55-71074c52e8aa.png)
It also loads wayy to much data! I think someone should change the code so that it only loads a X amount and averages the remaining in between. The X amount should probably not exceed the screen width in px, since it won't make sense.

**Comment by Fusseldieb** [comment]
Bump yet again. I just set up a dev environment on my local PC, restored my RPi3 database there and started the frontend to try and replicate my setup as close as possible. It loaded everything and **didn't** reload every 2 seconds, like I experienced and experience on the RP3. 
- It loads _once_ as it should and stays nicely.
- I also tried to set "Slow 3G" on Chrome to see it throttling the connection and ensuring it isn't hitting any timeout and retrying. It loaded quite slow, but also only loaded once, as it should.
- On both tests only one request to the HA API was sent. Good.
- The database had all devices in it, but obviously all were "dead", since it's only a backup of my real system, so no new states were fired whatsoever.
My conclusion is that it hasn't anything to do with the connection itself, but (very) probably with _something_ triggering a reload of that request, as seen in the "requests" screenshot above, since it's happening on the active system, but not on the "dead"-state system.
